person.rb

```diff
@@ -0,0 +1,22 @@
+class Person
+  attr_accessor :id, :name, :age
+
+  def initialize(id, name = 'Unknown', age = 0, parent_permission: true)
```

Comment:
- [ ] As per project requirements the Person Class should have a `Constructor with name, age, and parent_permission as parameter. name and parent_permission are optional and have default values of "Unknown" and true.` You have added id which is not specified in the project requirements. Also, only name and parent permission have default values. Kindly remove the default value for age.